doc/tutorial/MultiConfigModule.md

```diff
@@ -1,10 +1,150 @@
 # Declaring a multi-config module
+This tutorial shows how the will configuration can be splitted into two will files.
 
+## Module structure
+We are going to split our configuration file in two: an export and an import will files.
 
-#
-[Back to content](../README.md)
+```
+.
+├── fileToExport
+├── .ex.will.yml
+├── .im.will.yml
+```
+___
+
+## Creating the export configuration
+
+In a past tutorial, we saw how to export a module, review: [Exporting the module](Exporting.md).
+
+In this case, we are going to re-use that will config file, naming it `.ex.will.yml`.
+
+### Preparing the file for export:
+
+In this case we will export a single file, let´s create a `fileToExport` in the root directory.
+
+### Writing the export configuration:
+
+The final version of the `.ex.will.yml` file would be:
+
+<details>
+  <summary><u>Click to expand!</u></summary>
+
+```yaml
+about :
+
+  name : multi-config ex
+  description : "Multi-config module - export"
+  version : 0.0.0
+
+path :
+  out : '.'
+  fileToExport : './fileToExport'
+
+step  :
+  export.single :
+      inherit : export
+      tar : 0
+      export : path::fileToExport
+
+build :
+  export :
+      criterion :
+          default : 1
+          export : 1
+      steps :
+          - export.single
+```
+</details>
+
+
+## Creating the import build configuration
+
+In a past tutorial, we saw how to create a build configuration that runs npm to install dependencies, review: [Creating a build configuration](FirstBuild.md).
+
+In this case, we are going to re-use that will config file, naming it `.im.will.yml`. Moreover, we will modify it a bit, so it performs a simple shell command:
+
+Instead of installing the npm dependencies, it will output some text in the console. No additional files need to be prepared in this case.
+
+### Writing the import build configuration:
+
+The final version of the `.im.will.yml` file would be:
 
+<details>
+  <summary><u>Click to expand!</u></summary>
 
```

Comment:
Looks great, but also the goal is to show that sections defined in opposite will file( .ex.will.yml in this case ) are available everywhere( in '.im.will.yml . So there is no need to duplicate about section. Adjust import will file and add note about this behavior in text, please.